### ISSUE TITLE
[qualifiche] impossibile modificare dati

### ISSUE THREAD

**Issue by matthewmazzoleni** [CroceRossaItaliana/jorvik#342]
segnalano la non possibilità di aggiornare i dati di una qualifica (ad es. data scadenza)

**Comment by yakky** [comment]
@matthewmazzoleni  Questo mi sembra un comportamento voluto, immagino per semplificare la gestione della modifica dati nel caso di titoli che richiedano autorizzazioni etc.
E' comunque sempre possibile cancellarli e riaggiungerli
@AlfioEmanueleFresta mi confermi?

**Comment by AlfioEmanueleFresta** [comment]
@matthewmazzoleni @yakky esatto. Dopo l'approvazione, le qualifiche CRI non dovrebbero essere modificate. Questo perche' i dati della qualifica sono confermati dal Presidente/US. Le qualifiche CRI possono essere cancellate e aggiunte nuovamente. In questo modo, correttamente, viene richiesta una nuova approvazione dei dati.

L'alternative sarebbe permettere la modifica e richiedere una nuova conferma, ma e' tecnicamente un inferno (bisogna mantenere la vecchia qualifica fino ad approvazione, salvare da qualche parte la modifica proposta, tenere traccia dello storico, ecc.)

**Comment by luca-dex** [comment]
Sono d'accordo con @AlfioEmanueleFresta. Al limite si potrebbe scrivere da qualche parte (se già non è scritto) che le cose funzionano così.

**Comment by matthewmazzoleni** [comment]
@luca-dex @AlfioEmanueleFresta @yakky la richiesta in oggetto è stata fatta lato comitato. laddove ad esempio servisse aggiornare la scadenza di una patente la persona dovrebbe segnarsi tutti i dati su carta, cancellarla e reinserirla. questo perché ovviamente US o P non hanno sotto mano i documenti al momento dell'aggiornamento. vista comunque l'età dell'issue direi di reintavolare la cosa a fronte della gestione motorizzazione/accademia di formazione

**Comment by AlfioEmanueleFresta** [comment]
Confermo, anche perche' non mi sembrerebbe giusto modificare il record e andare a perdere lo storico. Anche simulando una modifica, il sistema dovrebbe andare a ricreare un nuovo record, copiando i dati rimasti uguali.

**Comment by yakky** [comment]
@matthewmazzoleni @AlfioEmanueleFresta ok, allora la metto in hold finché non è stata presa una decisione lato organizzativo
